### ISSUE TITLE
how to use the Client in non-async scenarios?

### ISSUE THREAD

**Issue by caphrim007** [stytchauth/stytch-python#204]
after the update to 9.x, my code that previously used to work, now raises RuntimeError exceptions because it is not running in an async context.

For example,

```
    client = Client(
        project_id=settings.stytch_project_id,
        secret=settings.stytch_secret,
        environment=settings.stytch_environment,
    )
```

in a flask application (without async), the following is raised

```
  File "/home/.../lib/python3.12/site-packages/stytch/consumer/client.py", line 42, in __init__
    super().__init__(
  File "/home/.../lib/python3.12/site-packages/stytch/core/client_base.py", line 24, in __init__
    self.async_client = AsyncClient(project_id, secret, session=async_session)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/.../lib/python3.12/site-packages/stytch/core/http/client.py", line 99, in __init__
    self._session = session or aiohttp.ClientSession()
                               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/.../lib/python3.12/site-packages/aiohttp/client.py", line 279, in __init__
    loop = get_running_loop(loop)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/.../lib/python3.12/site-packages/aiohttp/helpers.py", line 300, in get_running_loop
    loop = asyncio.get_event_loop()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/.../lib/python3.12/asyncio/events.py", line 698, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Dummy-2'.
2024-05-27T20:10:05Z {'REMOTE_ADDR': '127.0.0.1', 'REMOTE_PORT': '47724', 'HTTP_HOST': 'localhost:5000', (hidden keys: 33)} failed with RuntimeError
```

i downgraded to check if it worked in 8.7.0 and indeed it does.

The documentation here, for example, does not seem to work now in my flask app.

https://github.com/stytchauth/stytch-python?tab=readme-ov-file#example-usage

it does work, however if I just copy/pasta it into a test.py file and run that.

Any idea waht I'm missing here? or is this a bug?

**Comment by logan-stytch** [comment]
Hey @caphrim007, really sorry about this regression. I've just put up #205 which I think addresses this. By chance, do you have a minimally reproducible example of something that's failing so I could test it for you?

The test I added in #205 verifies that initializing the client will no longer attempt to retrieve the running event loop, at least, which should be sufficient.

**Comment by caphrim007** [comment]
lemme look. i have it baked into a flask application so it might be difficult to provide a minimal repro. maybe i can provide some hand-wavy examples and that might lead us towards a valid repro?

**Comment by logan-stytch** [comment]
@caphrim007 I just merged 9.5.1, so you could see if that fixed your issue.

**Comment by caphrim007** [comment]
checking back in here to say that 9.5.1 and subsequently 9.6.0 are working for me. thanks for resolving this!
